src/main/java/com/google/api/codegen/transformer/PathTemplateTransformer.java

```diff
@@ -97,8 +98,16 @@
     for (CollectionConfig collectionConfig : context.getCollectionConfigs()) {
       PathTemplateGetterFunctionView.Builder function = PathTemplateGetterFunctionView.newBuilder();
       function.name(namer.getPathTemplateNameGetter(collectionConfig));
+      function.resourceName(collectionConfig.getEntityName().replaceAll("_", " "));
```

Comment:
I would classify this as a casing conversion, and should be probably performed through NameFormatter.